src/app/constants/labels.js

```diff
@@ -1,18 +1,10 @@
 export const detailDefinitionLabels = {
-  // also provides sort for DefinitionList
   alt_titles: 'Alternative Titles',
-  medium: 'Medium',
-  series: 'Series',
   summary: 'Summary',
+  series: 'Series',
```

Comment:
@mwbenowitz haha I thought I had been seeing series - it's still here!

Reply:
whoops haha